ui.tests/test-module/specs/sites/captchaEmbed.runtime.spec.js

```diff
@@ -38,7 +38,11 @@ describe("Captcha In Sites Runtime Test", () => {
             expect(formContainer._model.items.length, "model and view elements match").to.equal(Object.keys(formContainer._fields).length);
             const [id, fieldView] = Object.entries(formContainer._fields)[0]
             const model = formContainer._model.getElement(id)
-            cy.get('#' + id + ' .cmp-adaptiveform-recaptcha__widget > div.g-recaptcha').should('exist');
+            cy.get('#' + id + ' .cmp-adaptiveform-recaptcha__widget > div.g-recaptcha').should('exist').then($iframe => {
```

Comment:
Can we also add one test case of captcha 400 error code in the embed scenario ?

Reply:
Added as requested